src/rameplayer/core/config.js

```diff
@@ -311,6 +311,230 @@
+            'DHCP range end': 'Конец диапазона DHCP',
+            'Time Configurations': 'Конфигурация времени',
+            'Device time (UTC)': 'Время устройства (UTC)',
+            'NTP server': 'NTP-сервер',
+            'Use IP address': 'Использовать IP-адрес',
+            'Configure date and time manually': 'Ручная установка даты и времени',
+            'Date:': 'Дата:',
+            'Time:': 'Время:',
+            'Factory reset': 'Заводские настройки',
+            'Restart device': 'Перезагрузить устройство',
+            // admin.js
+            DEVICE_HOSTNAME: 'Название узла устройства',
+            DEVICE_IP: 'IP-адрес устройства',
+            SUBNET_MASK: 'Маска подсети',
+            GATEWAY_IP: 'IP-адрес сетевого шлюза',
+            DNS_FIRST: 'DNS, основной',
+            DNS_SECOND: 'DNS, альтернативный',
+            DHCP_RANGE_START: 'Начало диапазона DHCP',
+            DHCP_RANGE_END: 'Конец диапазона DHCP',
+            DHCP_RANGE_DEF: 'Установка диапазона DHCP',
+            MANUAL_DATE: 'Ручная дата',
+            MANUAL_TIME: 'Ручное время',
+            NTP_SERVER_HOSTNAME: 'Название узла NTP-сервера',
+            ADMIN_SETTINGS_NOT_SAVED: 'Административные настройки не сохранены.',
+            CHECK_INSERTED_VALUES: 'Проверьте введенные данные',
+            INVALID_OPTIONAL_SETTINGS: 'Необязательные настройки возможно имеют неверные значения',
+            // settings.js
+            OPTION_SAVED: 'Параметр сохранен',
+            OPTION_AUTOPLAY_USB_DESC: 'Параметр автоматического проигрывания USB',
+            OPTION_ENABLED: 'вкл.',
+            OPTION_DISABLED: 'выкл.',
+            OPTION_SAVE_FAILED: 'Сохранение параметра неуспешно',
+            OPTION_AUTOPLAY_USB_FAILED_DESC: 'Сохранение праметра автоматического проигрывания USB неуспешно.',
+            LANGUAGE_SAVED: 'Язык сохранен.',
+            INVALID_CLUSTER_SETTINGS_DESC: 'IP-адрес нового устройства',
+            INVALID_SETTINGS: 'Неверные параметры',
+            NEW_ADDED:'Добавлено',
+            NEW_CLUSTER_UNIT_ADDED_DESC: 'Добавлено новое устройство кластера.',
+            // dataservice-provider.js
+            FIRMWARE_UPGRADED: 'Прошивка обновлена',
+            FIRMWARE_UPGRADED_DESC: 'Прошивка обновлена до версии $1.',
+            INCOMPATIBLE_VERSION: 'Несовместимая версия',
+            // statusservice.js
+            CONNECTION_ERROR: 'Ошибка соединения',
+            CONNECTION_ERROR_DESC: 'Не удалось соединится с устройством $1.',
+            RESTART_REQUIRED: 'Требуется перезагрузка',
+            RESTART_REQUIRED_DESC: 'Изменения требуют перезагрузку устройства. После перезагрузки откройте основную страницу устройства.', // jshint ignore:line
+            UPDATE_AVAILABLE: 'Доступно обновление',
+            // playlist.directive.js
+            UPDATE_AVAILABLE_DESC: 'Доступна более новая версия ПО.',
+            PLAYLIST_SYNC_REMOVED: 'Soittolistat $1 ja $2 eivät ole enää synkronoituna.',
```

Comment:
PLAYLIST_SYNC_REMOVED: 'Playlist $1 is not synchronized to playlist $2 anymore.',